nemo/collections/common/tokenizers/sentencepiece_tokenizer.py

```diff
@@ -43,6 +43,16 @@ def __init__(
             raise ValueError(f"model_path: {model_path} is invalid")
         self.tokenizer = sentencepiece.SentencePieceProcessor()
         self.tokenizer.Load(model_path)
+        # validate no token is negative
```

Comment:
Can we move this check inside the NMT model? This SentencePiece wrapper is common and is used by ASR as well and this condition might not be true for them.

Reply:
Done